apps/src/applab/commands.js

```diff
@@ -1076,7 +1076,7 @@ function setSize_(elementId, width, height) {
 
 applabCommands.setProperty = function (opts) {
   apiValidateDomIdExistence(opts, 'setProperty', 'id', opts.elementId, true);
-  apiValidateType(opts, 'setProperty', 'property', opts.property, 'string');
+  apiValidateType(opts, 'setProperty', 'property', opts.property, 'uistring');
```

Comment:
Please revert this line -- this affects the 'property' param whereas your description says you are trying to change how the 'value' param is treated.